pkg/Microsoft.NETCore.Platforms/runtime.json

```diff
@@ -7,6 +7,26 @@
             "#import": [ "base" ]
         },
 
+        "android": {
+            "#import": [ "any" ]
+        },
+        "android-arm": {
+	    "#import": [ "any" ]
+        },
+        "android-arm64": {
+	    "#import": [ "any" ]
```

Comment:
Well this is embarrassing. 
Removed tabs and re-pushed. If spacing still isn't correct I'll review using a different editor